build.gradle.kts

```diff
@@ -108,8 +109,22 @@ dependencies {
         exclude(group = "log4j")
     }
     implementation("org.apache.kafka:kafka_2.13:${Versions.kafkaVersion}", excludeLog4j)
+    constraints {
+        implementation("org.scala-lang:scala-library") {
+            because("org.apache.kafka:kafka_2.13:${Versions.kafkaVersion} -> https://www.cve.org/CVERecord?id=CVE-2022-36944")
+            version {
+                require(Versions.scala)
+            }
+        }
+    }
     testImplementation("no.nav:kafka-embedded-env:${Versions.kafkaEmbeddedVersion}", excludeLog4j)
     constraints {
+        implementation("org.yaml:snakeyaml") {
+            because("no.nav:kafka-embedded-env:${Versions.kafkaEmbeddedVersion} -> https://advisory.checkmarx.net/advisory/vulnerability/CVE-2022-25857/")
+            version {
+                require("1.31")
```

Comment:
Burde denne også få en `version.snakeyaml`, når det er det for Scala?

Reply:
Tja, kan godt gjøre det, men disse kan jo fort bli borte igjen når man får en ny versjon av kafka-embedded-env (i dette tilfellet).